libraries/AP_Baro/AP_Baro_DPS280.cpp

```diff
@@ -126,6 +126,7 @@ bool AP_Baro_DPS280::init()
         return false;
     }
 
+    dev->set_read_flag(0x80);
```

Comment:
Should be used only if  sensor on spi bus.

Reply:
its harmless on i2c. We normally put it in unconditionally, so it works for both bus types